app/controllers/optIn/CheckYourAnswersController.scala

```diff
@@ -29,11 +29,12 @@ import play.api.mvc._
 import services.DateService
 import services.optIn.OptInService
 import utils.AuthenticatorPredicate
-import views.html.optIn.CheckYourAnswersView
+import views.html.optIn.{CheckYourAnswersView, ChooseTaxYearView}
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
+/* todo will be fully implemented in MISUV-8006 */
```

Comment:
Hasn't MISUV-8006 (Check your answers page) been done? Do you need to merge main?

Reply:
done